src/cookies.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import {Services} from './services';
```

Comment:
I don't know if this is safe. When is the platform code first installed, and where is the first `setCookie` call?

Reply:
I'm not sure it is safe. The platform service is installed as one of the first things in the first startup chunk here: https://github.com/ampproject/amphtml/blob/1c1e722ad99fc5352e5519b28cdb0db99854e2fd/src/amp.js#L91-L94

The first call to `setCookie` on the page is when the user clicks button to opt in/out of canary (or RC). Since the scripts use defer, we can be sure that v0 runs on this page prior to the code that listens to the button click is registered, but since the platform service install is in a startup chunk, I'm pretty sure there is a window where the user could click the button first and have the event listener present, before the macro task for the first startup chunk is run.

Reply:
It's not the experiments page that I'm worried about, but ads/analytics on a regular amp page that calls `setCookie`.

Reply:
So since this is fixed in iOS 13, and I think only the experiments page needs `samesite=None`, how does removing the platform check sound? I'll update the PR with that for now.